app/src/main/java/com/teamwss/websoso/ui/memoPlain/MemoPlainActivity.kt

```diff
@@ -0,0 +1,121 @@
+package com.teamwss.websoso.ui.memoPlain
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.view.WindowManager
+import androidx.activity.viewModels
+import androidx.appcompat.app.AppCompatActivity
+import com.teamwss.websoso.databinding.ActivityMemoPlainBinding
+import com.teamwss.websoso.ui.memoWrite.MemoWriteActivity
+import kotlin.properties.Delegates
+
+class MemoPlainActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityMemoPlainBinding
+    private val memoPlainViewModel: MemoPlainViewModel by viewModels()
+    private var memoId by Delegates.notNull<Long>()
+
+    private lateinit var memoContent: String
+    private lateinit var userNovelTitle: String
+    private lateinit var userNovelAuthor: String
+    private lateinit var userNovelImage: String
+    private val dialogMemoDelete: DialogMemoDelete by lazy {
+        DialogMemoDelete(::finish)
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityMemoPlainBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        memoId = intent.getLongExtra("memoId", -1)
+        memoPlainViewModel.updateMemoId(memoId)
+        memoPlainViewModel.getMemo(memoId)
+
+        setTranslucentOnStatusBar()
+        setupLifecycleOwner()
+        setupDataBinding()
+        getUserNovelDataToIntent()
+        observeMemoId()
+        observeMemoContent()
+        onClickMemoPlainCancelButton()
+        onClickMemoDeleteButton()
+        onClickMemoEditButton()
+    }
+
+    private fun setTranslucentOnStatusBar() {
+        window.setFlags(
+            WindowManager.LayoutParams.FLAG_LAYOUT_NO_LIMITS,
+            WindowManager.LayoutParams.FLAG_LAYOUT_NO_LIMITS
+        )
+    }
+
+    private fun setupLifecycleOwner() {
+        binding.lifecycleOwner = this
+    }
+
+    private fun setupDataBinding() {
+        binding.memoPlainViewModel = memoPlainViewModel
+    }
+
+    private fun getUserNovelDataToIntent() {
+        userNovelImage = intent.getStringExtra("userNovelImage").toString()
+        userNovelTitle = intent.getStringExtra("userNovelTitle").toString()
+        userNovelAuthor = intent.getStringExtra("userNovelAuthor").toString()
+    }
```

Comment:
저는 보통 모델로 묶어서
프로퍼티쪽에

```suggestion
    private val userNovel: UserNovel by lazy {
        intent.getParcelableExtra(USER_NOVEL)
    }
```
요로코롬 할듯